pkg/virt-v2v/customize/scripts/windows/9100_cleanup_vmware.ps1

```
@@ -0,0 +1,412 @@
+    $result = & $RegExe delete $KeyPath /f 2>&1 | Out-String
+    if ($LASTEXITCODE -eq 0) {
+        Log "[DEL KEY] $KeyPath"
+        $script:regDeleted++
+    } else {
+        Log "[WARNING] reg.exe delete failed: $KeyPath - $result"
+        $script:regErrors++
+    }
+}
+
+# Delete a single registry value via reg.exe.
+function Delete-RegValue {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Delete-RegValue' uses an unapproved verb.

(PSUseApprovedVerbs)